### ISSUE TITLE
Error when not inside Mingo

### ISSUE THREAD

**Issue by Christilut** [mingo-app/mingo#278]
![image](https://user-images.githubusercontent.com/3471363/126153277-983beffb-d665-4b66-b0b1-38db589be9a6.png)

I closed Mingo but I get this error when I press `CTRL+R`.

Upon closer inspection it looks like Mingo is still running:

![image](https://user-images.githubusercontent.com/3471363/126153367-01ca2d06-d72a-4ad8-9558-f1646c2dfc41.png)

Although I don't see it anywhere.

The error happens in any program and doesn't stop until I kill mingo.

Killing mingo is also quite difficult:
![image](https://user-images.githubusercontent.com/3471363/126153546-656f8153-26d2-4ee5-9eed-0be426094e38.png)

This is an admin shell. Killing it by choosing "End task" in the task manager does not work.

With `/f` it does work.

**Comment by Christilut** [comment]
Same thing happened again. Seems to happen quite often

**Comment by Christilut** [comment]
Happening every time I close mingo it seems. Getting so far that I can't really use other programs where I press CTRL+R without getting an error

**Comment by tothradoslav** [comment]
I have just released a pre-release, please check out all the "ready for next release" tasks. Thank you!

https://github.com/mingo-app/mingo/releases/tag/v1.4.5-rc.1

**Comment by tothradoslav** [comment]
it's v1.4.5 now :)

**Comment by Christilut** [comment]
This is happening again with 1.5.0

**Comment by tothradoslav** [comment]
Is this on a mac or windows? Cannot reproduce on a mac. Even if I keep the app running with zero windows opened (as per #293), I can still use CMD+R in other apps properly. 

Could you please send more details about how to reproduce?

**Comment by Christilut** [comment]
Windows.
I can't reproduce it right now. I did notice Mingo.exe was running ~5 times in the task manager (no visible Mingo windows) and I had to kill those processes before CTRL+R worked again.

**Comment by Christilut** [comment]
Happens every day. Only using Mingo for a bit, not doing anything special.

Killing it in the task manager is near impossible, it just doesn't seem to respond to End Task.

Command line works but still hard.

![image](https://user-images.githubusercontent.com/3471363/141095348-23ee4391-2abf-4da2-9000-742c4f62570b.png)

I would make sure to fix this because for a user this all seems very dubious. Random errors and an unkillable process scares people...

**Comment by tothradoslav** [comment]
Thanks, going to look into this. Does this only happen in Windows? So far, I could not reproduce it. Maybe there's something you (I) have to do before this starts happening... If you have multiple processes running, does it mean you have opened and closed Mingo app several times?

**Comment by Christilut** [comment]
Yes Windows only.

What I've noticed is that it doesn't happen when I just start Mingo once and closed it (for testing purposes). I always only notice it after I've used Mingo and left it in the background for a while (5+ minutes or so).

Maybe it has something to do with the deeplinks, as they open new Mingo instances each time (that I then close because I don't want multiple instances and the deeplink was lost anyway). 

Sometimes there are 6 or so Mingo processes running, all very hard to kill
